packages/EasyNotification/src/Queue/SqsQueueTransport.php

```diff
@@ -22,10 +23,19 @@ public function __construct(SqsClient $sqs)
 
     public function send(QueueMessageInterface $queueMessage): void
     {
+        $queueUrl = $queueMessage->getQueueUrl();
+
+        if (\str_ends_with($queueUrl, '.fifo')) {
+            $messageDeduplicationId = (string)Uuid::v4();
+            $messageGroupId = (string)Uuid::v4();
+        }
```

Comment:
I think it's ok not to complicate things in this package as we are the only ones to use it really, but we need to at least required the `symfony/uid` package in composer so we ensure the `Uuid` class is available 😄